### ISSUE TITLE
CloneFunction() returns same value on all samples during evaluation

### ISSUE THREAD

**Issue by DerMascha** [Microsoft/CNTK#1669]
I have a model that combines the (weighted) output of two other pre-trained models into a binary classifier:

```
labels = Input {labelDim}
color = Input {imageShape}
depth = Input {imageShape}	

colorNet = BS.Network.Load("$outputDir$/Models/Separate_LowRes/color")
depthNet = BS.Network.Load("$outputDir$/Models/Separate_LowRes/depth")
	
colorNetwork = BS.Network.CloneFunction(colorNet.color, colorNet.cout)
depthNetwork = BS.Network.CloneFunction(depthNet.depth, depthNet.dout)
	
# apply model to features
spliced = Splice(colorNetwork(color):depthNetwork(depth), axis=1)
mod = ParameterTensor{1:2, initValue=1.0, learningRateMultiplier=10.0}
	
output = Times(mod, spliced) #scalar
pred = Tanh(output)

# loss computation
ce = SquareError(pred, labels)
```

While everything works fine during training (average accuracy ~97%), when I try to evaluate the network using the **test** or **cv** command, I get an accuracy of ~50%, which means the network is randomly guessing.

I dumped the output of the various layers and found that the two imported networks calculate the first sample correctly, but then return the same value for every consecutive iteration (the input, however varies).
This is the case even if I evaluate against the training set (which produced correct results during training). Each imported model was evaluated against the same test-set and gave back the correct results.
It also doesn't matter if cloned network's parameters are _learnable_ or _constant_.

So it seems like CloneFunction() isn't reevaluated for each evaluation step.

**Comment by cha-zhang** [comment]
@DerMascha , sorry for the very late response. Has this issue be resolved with the latest CNTK release?

**Comment by cha-zhang** [comment]
I'm closing this issue due to lack of response. Please reopen the issue if needed. Thanks!
